runtime/src/main/codestarts/quarkus/quakus-hilla-maven-codestart/codestart.yml

```diff
@@ -0,0 +1,20 @@
+name: quarkus-hilla-maven-codestart
+ref: quarkus-hilla-maven-starter
+type: code
+tags:
+  - extension-codestart
+  - maven-only
+
+metadata:
+  title: Quarkus-Hilla starter
+  description: Start to code with the Quarkus-Hilla extension.
+  related-guide-section: https://github.com/mcollovati/quarkus-hilla/wiki/QuickStart
+language:
+  base:
+    shared-data:
+      boms:
+        - groupId: dev.hilla
+          artifactId: hilla-bom
+          version: "\${hilla.version}"
```

Comment:
Why using the escaped version here an below the unescaped (and missing quotes).
From where comes the hilla.version? Is this the default version from the pom and can be overwritten?

Reply:
It is using the escape because the result in the produced pom file should be `<version>${hilla.version}</version>`, not the expanded value.
The other value should instead be filtered to get the actual hilla version defined in the extension parent pom, so the template will produce the property `<hilla.version>x.y.z</hilla.version>` in the pom file.

Reply:
Get it, but wouldn't it be easier to place `${hilla.version}` directly into the pom? Or is it then also filtered out?

Reply:
If we put the dependency in the pom template, it will be put below the quarkus-bom and this may lead with some transitive dependencies issues.
However, if we add it through the `boms` key in the codestart descriptor the hilla-bom will be placed before the quarkus-bom